tests/formspree_test_case.py

```diff
@@ -0,0 +1,34 @@
+
+import os
+import fakeredis
+
+from flask.ext.testing import TestCase
+import formspree
+from formspree import create_app
+from formspree import settings
+from formspree.app import DB, REDIS
+
+
+class FormspreeTestCase(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        ''' configures app for testing '''
+        formspree.app.get_redis = lambda : fakeredis.FakeStrictRedis()
```

Comment:
Just saying, it's not normal to add collateral, when it catches you it's normally long hours debugging to discover what you thought was redis is actually another class, hehe.